manim/camera/multi_camera.py

```diff
@@ -12,15 +12,15 @@ class MultiCamera(MovingCamera):
 
     def __init__(
         self,
-        image_mobjects_from_cameras=None,
-        allow_cameras_to_capture_their_own_display=False,
+        image_mobjects_from_cameras: any = None,  # TODO: Type this.
```

Comment:
I guess that's ImageMobject type (not sure)

Reply:
It was causing issues with my typing this.

Reply:
Typing this caused errors, I'll let someone else deal with it.